src/coins.c

```diff
@@ -1,14 +1,15 @@
+#include "global.h"
```

Comment:
This is already included by the compiler

Reply:
sure it is, but it causes errors in the IDE if I don't include it

Reply:
Then get a better IDE or push without the change smh

Reply:
genuinely, all c files should include global.h imo

Reply:
(exception to this being libs ofc)

Reply:
ok, remove it here so we can merge pls

Reply:
this isn't a lib tho??? stuff in global.h is referenced in the file?

Reply:
sry was in response to your second comment, we're not gonna include global.h in everything